File: cs/Azyobuzi.TwitterUrlExtractor/EntityInfo.cs
namespace Azyobuzi.TwitterUrlExtractor
{
    public struct EntityInfo
    {
        public int StartIndex { get; set; }
        public int Length { get; set; }

        public EntityInfo(int startIndex, int length)
        {
            this.StartIndex = startIndex;
            this.Length = length;
        }
    }
}

File: java/Azyobuzi.TwitterUrlExtractor/src/main/java/net/azyobuzi/twitterurlextractor/EntityInfo.java
package net.azyobuzi.twitterurlextractor;

public class EntityInfo {
    private int startIndex;
    private int length;

    public int getStartIndex() {
        return this.startIndex;
    }

    public int getLength() {
        return this.length;
    }

    public EntityInfo(int startIndex, int length) {
        this.startIndex = startIndex;
        this.length = length;
    }
}
